packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -277,7 +277,7 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
         _setFramesEnabledState(true);
         break;
       case AppLifecycleState.paused:
-      case AppLifecycleState.suspending:
+      case AppLifecycleState.detached:
```

Comment:
worth adding something to the tests on this side in dev/integration_tests as well now that we have integration tests on the framework. Perhaps in the ios_add2app test.

Reply:
+1

Reply:
done

Reply:
I can't run this:
```
lib/main.dart:36:23: Error: Getter not found: 'detached'.
    AppLifecycleState.detached,
                      ^^^^^^^^
../../../../packages/flutter/lib/src/scheduler/binding.dart:278:30: Error: Getter not found: 'detached'.
      case AppLifecycleState.detached:
                             ^^^^^^^^
../../../../packages/flutter/lib/src/scheduler/binding.dart:298:34: Error: Getter not found: 'detached'.
        return AppLifecycleState.detached;
```
Does it need to be added to `AppLifecycleState`?

Reply:
Never mind, I see it needs to be on https://github.com/flutter/engine/pull/11913.

Reply:
I compile the engine and put --local-engine=host_debug_unopt in https://github.com/flutter/flutter/pull/39945/files#diff-98f8467c300c0f2c4e14432a072a9306R8